src/context/update.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { https } from 'follow-redirects'
+import fs from 'fs'
+import { tmpNameSync } from 'tmp'
+import { commands, ProgressLocation, Uri, window } from 'vscode'
+import '../shared/extension'
+
+// Usage:
+// commands.registerCommand('sarif.updateExtension', async () => await update())
+export async function update() {
+	const fileName = tmpNameSync({ postfix: '.vsix' })
+	const stream = fs.createWriteStream(fileName)
+	const url = new URL('https://github.com/jeffersonking/sarif-vscode/releases/download/Latest/sarif-vscode-0.0.0.vsix')
```

Comment:
>https://github.com/jeffersonking/sarif-vscode/releases/download/Latest/sarif-vscode-0.0.0.vsix [](start = 22, length = 94)

?

Reply:
Custom update functionality that was being used to provide "insiders" updates. `update()` is not current being used, but may become useful to my team (SARIF) in certain cases. We do not share your same Windows pipeline and the release process is thus a bit different.